### ISSUE TITLE
I'm getting 4 of every console message

### ISSUE THREAD

**Issue by xrx101** [inspectdev/inspect-issues#44]
- inspect-app version: 0.6.3
- OS ARCH VERSION: win32 x64 10.0.19042
- Electron: 11.2.1
- Language: undefined

Steps to Reproduce:

1. Plug device in
2. "Inspect" it - open in Console
3. Most messsages get collapsed and show once with a 4 in a blue circle, but when the log includes an object reference, you get 4 different lines.  There are no exceptions, it's 4 of everything.

**Comment by auchenberg** [comment]
@xrx101 Thanks for reporting this. Would it be possible for you share an example of code triggering this behavior?

**Comment by xrx101** [comment]
So sorry for not replying earlier as been away and switched off entirely (highly recommended).

After posting, I was at times plugging an iDevice in and inspecting and getting between 4 and 8 message duplications for consoles that should have only ever been produced once.  It wasn't consistent at all, but 4 duplications happened more often than not.  Even after rebooting a few times, I would get similar results.

Unfortunately I can't share the code that produces it because it is private IP, but I can tell you that it is a php/javascript application and for all testing of older iOS devices, we've used the remotedebug_ios_webkit_adapter and have never seen duplicate console messages.

Having returned from leave, I've just plugged in a number of devices and am not able to reproduce the issue. The code base hasn't changed at all and the version of inspect has not changed either.

Very strange, but at this point I think it maybe an isolated incident related to my setup.  I'll raise a flag if it starts happening again, but I've been at it for about 4 hours now and seems fine.
